### ISSUE TITLE
Proxy Service does not append annotations defined in ProxyFleet

### ISSUE THREAD

**Issue by oliverjanka** [jeremylvln/Shulker#303]
### What happened?

This is the Proxy configuration:
```yaml
apiVersion: shulkermc.io/v1alpha1
kind: ProxyFleet
metadata:
  name: public
spec:
  clusterRef:
    name: minecraft
  replicas: 1
  service:
    type: LoadBalancer
    externalTrafficPolicy: Local
    annotations:
      metallb.universe.tf/loadBalancerIPs: 192.168.1.253
  template:
    spec:
      version:
        channel: Velocity
        name: latest
      config: {}
```
This is the resulting service:
```yaml
apiVersion: v1
kind: Service
metadata:
  annotations:
    metallb.universe.tf/ip-allocated-from-pool: default-pool
...
```
This is what I would expect:
```yaml
apiVersion: v1
kind: Service
metadata:
  annotations:
    metallb.universe.tf/ip-allocated-from-pool: default-pool
    metallb.universe.tf/loadBalancerIPs: 192.168.1.253
...
```

Thank you very much for your help and also this great project!

### What components are involved in your issue?

Operator

### Version

0.4.0

### Kubernetes Version

1.28

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jeremylvln** [comment]
Hi @oliverjanka!

Sorry that you encountered an issue using Shulker :(

I just merged this fix recently, and I don't have published a release yet, I was waiting to merge another feature before doing that. However, if you pin the version of Shulker to `next` (using the Docker image tag `next`), you should be able to use this fix right away. I will work to merge the last feature today so a proper release will be done just after.

Thanks!

**Comment by jeremylvln** [comment]
This feature is now released, check version 0.5.0 :)
